java/pom.xml

```diff
@@ -46,7 +46,7 @@
     <errorprone.javac.version>9+181-r4173-1</errorprone.javac.version>
     <error_prone_core.version>2.24.0</error_prone_core.version>
     <maven-compiler-plugin.version>3.12.1</maven-compiler-plugin.version>
-    <mockito.core.version>5.5.0</mockito.core.version>
+    <mockito.core.version>5.11.0</mockito.core.version>
```

Comment:
This is the latest mockito version and is used for Java 11+ (Java 8 is pinned lower). Tests were failing for Java 22 on mockito 5.5.0.